editor/src/components/navigator/left-pane/left-pane-utils.ts

```diff
@@ -0,0 +1,17 @@
+import type { ElementPath } from '../../../core/shared/project-file-types'
+import { LeftMenuTab } from '../../editor/store/editor-state'
+
+export function nextSelectedTab(
+  currentLeftMenuTab: LeftMenuTab,
+  newSelectedPaths: ElementPath[],
+): LeftMenuTab {
+  if (newSelectedPaths.length === 0) {
+    return currentLeftMenuTab
+  }
+
+  if (currentLeftMenuTab === LeftMenuTab.Pages) {
+    return LeftMenuTab.Pages // same as LeftMenuTab.Pages
```

Comment:
can confirm it is indeed same as LeftMenuTab.Pages :)

Reply:
technically true https://github.com/concrete-utopia/utopia/pull/5271/commits/31e85e2eeaa2d76a8f3b7e36e14d8f1ab03655e2